.github/workflows/mobile_ci.yml

```diff
@@ -25,11 +25,23 @@ jobs:
           ENV_FILE: ${{ secrets.ENV }}
         run: echo "$ENV_FILE" > ".env"
 
+      - name: Clean npm cache
+        run: npm cache clean --force
+
+      - name: Delete previous node modules
+        run: rm -rf node_modules
+
       - name: Install dependencies
         run: npm install
 
+      - name: Clear ESLint cache
+        run: rm -f .eslintcache
+
+      - name: Clean TypeScript cache
+        run: rm -rf ./node_modules/.cache/typescript
+
```

Comment:
For learning purposes: Why do we have to clean the cache? I thought each time it runs, it's a different job and have fresh cache?

Reply:
Well, that _was_ the assumption but this was implemented due to an issue during CI.

So basically, on the intial CI, we had an unused variable in the code and so it failed. Then, I removed that variable and the tests re-ran. But it kept failing with the same output message (that is, the unused variable exists even tho it doesnt). So what you see above is me trying to clear any caches so that ESLint doesn't keep hallucinating the error. But eventually what fixed it was something different. These two didn't do the trick.

However, I left them there because from what I understand, there is definitely some caching going on during our build process and i put them there so we're not blindsided by caching issues in the future.

Reply:
I'm gonna remove these and check if our CI is successful. Resolving the conversation for now.

Reply:
Oh I see, such a pain with the CI